src/js/api.js

```diff
@@ -1,26 +1,37 @@
 import fetch from 'isomorphic-fetch'
 
-function sendFetch(suffix, options = {method: 'GET'}){
-  var url = makeUrl(parseLocation(), suffix);
+let accessToken
 
+function sendFetch(suffix, options = {method: 'GET'}){
+  var location = options.noParse ? {} : parseLocation();
```

Comment:
So `<server>/1/2017` can be mined for the institution id and the filing period to use in api requests, but a request for individual institutions from the homepage or institutions page can't use anything in the url (this is why the function signature includes the `id` param for `getInstitution`)